src/Symfony/Component/HttpKernel/Tests/Fixtures/BundleDependencies/BundleCDependenciesBA.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies;
+
+use Symfony\Component\HttpKernel\Bundle\BundleDependenciesInterface;
+
+class BundleCDependenciesBA implements BundleDependenciesInterface
+{
+    public function getBundleDependencies()
+    {
+        return array(
+            'Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies\BundleBDependenciesA',
```

Comment:
sure .. but what would be the advantage? or are you just proposing a possible new format with migration path in case we leave it out today and want to add optional deps later?

just seems like a more complicated structure. plus an assoc array implicitly takes care of duplicates. then again if one ends up having so many dependencies that accidental duplicates are a risk, one might be over using the feature :)